manifests/config.pp

```diff
@@ -97,4 +97,22 @@
       refreshonly => true,
     }
   }
+
+  if $pulp::lazy_enabled {
+    class { '::squid3':
```

Comment:
Is squid the Pulp supported option for lazy sync? @cristifalcas would you need a parameter here such as `manage_squid` or is this setup and configuration of squid if lazy sync is enabled OK?